web/src/components/JSMpegPlayer.jsx

```diff
@@ -7,6 +7,20 @@ export default function JSMpegPlayer({ camera, width, height }) {
   const playerRef = useRef();
   const url = `${baseUrl.replace(/^http/, 'ws')}/live/${camera}`
 
+  let wakeLock = null;
```

Comment:
hi @thburch , this might not be working as expected, since React (though not sure about preact) will run this function on each re-render cycle and `wakeLock` will be set to null. What I would do is to:
```
const [wakeLock, setWakeLock] = useState(null)
```
and then use `setWakeLock` to set the value